plugins/proxy-backend/config.d.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface Config {
+  /**
+   * A list of forwarding-proxies. Each key is a route to match,
+   * below the prefix that the proxy plugin is mounted on. I must
+   * start with a '/'.
+   */
+  proxy: { [key: string]: string | ProxyConfig };
```

Comment:
not sure if we want to make it required

Reply:
does the backend start without the proxy config?

Reply:
It does start without a config https://github.com/backstage/backstage/blob/a081f9fac899f8965759684a446922734a57a658/plugins/proxy-backend/src/service/router.ts#L128

That seems to show that we should make it also optional in the schema. Doesn't it?